src/components/base/RegisterForm.tsx

```diff
@@ -0,0 +1,118 @@
+import InputField from './Input'
```

Comment:
Use `Input` instead of `InputField` so the name matches it's definition

Reply:
Changed name to Input